.github/workflows/task-test.yml

```diff
@@ -172,11 +201,7 @@ jobs:
         if: >
           steps.node20.outputs.supported == 'false' &&
           (steps.unit_tests.outcome == 'failure' || steps.standalone_tests.outcome == 'failure' || steps.coordinator_tests.outcome == 'failure')
-        uses: actions/upload-artifact@v3
-        with:
-          name: Test Logs ${{ steps.artifact-names.outputs.name }}
-          path: tests/**/logs/*.log*
-          if-no-files-found: ignore
+        run: echo "Currently not available..."
```

Comment:
Can do it in a separate PR (maybe even just a `grep -C` for some `[FAIL]` etc. tokens